tests/load/user_flow.js

```diff
@@ -1,12 +1,15 @@
 import { cart_data } from "./rpcs/cartdata.js";
 import { shipping_and_tax } from "./rpcs/shippingtax.js";
 import { create_order } from "./rpcs/create_order.js";
-import { check } from "k6";
+import { check, sleep } from "k6";
 import { Rate } from "k6/metrics";
 import { create_header } from "./config.js"; 
 
 const errorRate = new Rate( "errors" );
 
+// Need this variable when simulating less than 1 user per second 
+const SLEEP = __ENV.SLEEP ? parseInt(__ENV.SLEEP) : 0;
```

Comment:
What if the env variable is set to an invalid string by accident? Can you make sure it defaults to 0 if it's invalid?

Reply:
Sounds good, updated - tested with invalid strings and defaults to 0 if string is not a number